homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -35,6 +48,39 @@
 
 SUCCESS = ["ok"]
 
+GATEWAY_SENSOR_TYPES = {
+    "temperature": [TEMP_CELSIUS, None, DEVICE_CLASS_TEMPERATURE],
+    "humidity": [UNIT_PERCENTAGE, None, DEVICE_CLASS_HUMIDITY],
+    "pressure": ["hPa", None, DEVICE_CLASS_PRESSURE],
+}
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the Xiaomi sensor from a config entry."""
+    entities = []
+
+    # Gateway subdevices
+    if config_entry.data[CONF_FLOW_TYPE] == CONF_GATEWAY:
+        gateway = hass.data[DOMAIN][config_entry.entry_id]
+        subdevices = gateway.devices
+        for subdevice in subdevices:
+            if subdevice.device_type == "AqaraHT":
```

Comment:
no subdevice.device_type is a string, subdevice.type is the enum, but I will change it to the enum, just did not want to import the enum from miio for this, thought this would be easier.

Reply:
changed it